apisix/discovery/consul_kv.lua

```diff
@@ -244,7 +244,7 @@ end
 local function read_dump_srvs()
     local data, err = util.read_file(dump_params.path)
     if not data then
-        log.warn("read dump file get error: ", err)
+        log.notice("read dump file get error: ", err)
```

Comment:
I think we can check if the file exists first via 
https://refined-github-html-preview.kidonng.workers.dev/lunarmodules/Penlight/raw/master/docs/libraries/pl.path.html#exists

If not exists, log a notice message.

And then try to read it.

Reply:
Y

Reply:
@yongboy 
We just add the heavy library in the master branch: https://github.com/apache/apisix/blob/1064e2d4b7edcc6f590ca69dc34a100ae6bbc12c/rockspec/apisix-master-0.rockspec#L68.
Would be fine if we can find another usage of this library.

Reply:
I think we don't need toimport the heavy library `penlight` at the moment. It is so heavy!
If another one needs it really, then we can change the code later.